client/wfm/src/service/WfmService.ts

```diff
@@ -0,0 +1,241 @@
+import * as Promise from 'bluebird';
+import * as _ from 'lodash';
+import * as shortid from 'shortid';
+import { WorkOrderResult } from '../result/WorkOrderResult';
+import { STATUS } from '../status';
+import { Step } from '../step/Step';
+import { WorkFlow } from '../workflow/WorkFlow';
+import { WorkOrder } from '../workorder/WorkOrder';
+import { DataService } from './DataService';
+import { ResultService } from './ResultService';
+import { UserService } from './UserService';
+
+interface CompleteStepParams {
```

Comment:
They're the parameters for the function: https://github.com/feedhenry-raincatcher/raincatcher-core/blob/RAINCATCH-1121-corewfm/client/wfm/src/service/WfmService.ts#L127, not a return value, only here for documentation.

Alternatively we can change the function to receive the 3 params separately instead of through an object.